lib/services/role.go

```diff
@@ -993,6 +1010,16 @@ func MatchLogin(selectors []string, login string) (bool, string) {
 	return false, fmt.Sprintf("no match, role selectors %v, login: %v", selectors, login)
 }
 
+// MatchAWSRoleARN returns true if provided role ARN matches selectors.
+func MatchAWSRoleARN(selectors []string, roleARN string) (bool, string) {
+	for _, l := range selectors {
+		if l == roleARN {
+			return true, "matched"
+		}
+	}
+	return false, fmt.Sprintf("no match, role selectors %v, role ARN: %v", selectors, roleARN)
```

Comment:
Is this 2nd return parameter used at all?

Reply:
Yeah, it's printed in some CheckXXX methods for easier troubleshooting IIRC, same as for other matchers.